### PR TITLE
Enable mutiple sgs in AWS

### PR DESCRIPTION
**Pull request by Razz**

### CODE REVIEW

File: virtualmachine/aws/vm.go [hunk 1]
```diff
@@ -81,9 +81,9 @@ type VM struct {
 	KeepRootVolumeOnDestroy      bool
 	DeleteNonRootVolumeOnDestroy bool
 
-	VPC           string
-	Subnet        string
-	SecurityGroup string
+	VPC            string
+	Subnet         string
+	SecurityGroups string
```
Comment (lilirui) [inline virtualmachine/aws/vm.go]: Prefer to change the data type to the array of string.

**Comment by variadico** [comment]
Change LGTM. But, it does break the API.

**Comment by Razz** [comment]
Could split into two pulls, one using the [string implementation](https://github.com/apcera/libretto/pull/108/commits/0a4de5d2513bcf93520aa151d54be6a40b1c8346), then whenever you all version bump, pull in the full PR.

File: virtualmachine/aws/vm.go (hunk 1 above)
Comment (zquestz) [inline virtualmachine/aws/vm.go]: Agreed, this should be pulled in with the array implementation, and we should bump the version in this PR, so it merges with the version change.

**Comment by zquestz** [comment]
Also need to update README.md and any sample code so it is accurate with this change.

**Comment by variadico** [comment]
Gonna merge and bump the version to 1.0.0.

**Pull request merged**